workflow/rocoto/tasks.py

```diff
@@ -223,7 +223,10 @@ def get_resource(self, task_name):
             else:
                 native += ':shared'
         elif scheduler in ['slurm']:
-            native = '--export=NONE'
+            if task_config.get('is_exclusive', False):
+                native = '--export=ALL --exclusive'
```

Comment:
Why is `--export=ALL` required for exclusive?
This is going to break every HPC

Reply:
Let me try remove --export=ALL.

Reply:
tested that remove "--export=ALL" from "native" when is_exclusive is set True in config.resource.AWSPW works fine.